src/app/containers/App/index.test.jsx

```diff
@@ -28,24 +41,21 @@ describe('ClientApp', () => {
     );
   });
 
+  it('should render App component', () => {
```

Comment:
Why was this added?

Reply:
It was always there. It used to be on line 46.